wagtail/admin/tests/test_workflows.py

```diff
@@ -1414,12 +1414,20 @@ def test_email_headers(self):
         # Submit
         self.post("submit")
 
-        msg_headers = set(mail.outbox[0].message().items())
+        message = mail.outbox[0].message()
+        msg_headers = set(message.items())
         headers = {("Auto-Submitted", "auto-generated")}
         self.assertTrue(
             headers.issubset(msg_headers),
             msg="Message is missing the Auto-Submitted header.",
         )
+        self.assertTrue(message.is_multipart())
+
+    @override_settings(WAGTAILADMIN_NOTIFICATION_USE_HTML=False)
```

Comment:
I'm thinking it's worth subclassing the whole class with this decorator, in case we miss anything else.